packages/nodes/table/src/components/TableCellElement/TableCellElementResizable.tsx

```diff
@@ -46,11 +51,23 @@ export const useTableCellElementResizableProps = ({
     ELEMENT_TABLE
   );
   const element = useElement();
+  const tableElement = useElement(ELEMENT_TABLE);
+  const path = useMemo(() => findNodePath(editor, element)!, [editor, element]);
```

Comment:
opti: can be moved into the callback, we generally don't use `useMemo` when it's used only in callbacks

Reply:
Can you explain which callback this `useMemo` can be moved into? The only reason I'm using it here is because I'm wary of doing any computations at the top level of a render function.